cmd/kubeadm/app/phases/addons/proxy/manifests.go

```diff
@@ -75,6 +75,8 @@ spec:
         command:
         - /usr/local/bin/kube-proxy
         - --kubeconfig=/var/lib/kube-proxy/kubeconfig.conf
+        {{ .KubeProxyMode}}
+        {{ if .KubeProxyMode }}- --feature-gates=SupportIPVSProxyMode=true{{ end }}
```

Comment:
isn't that feature gate needed only in case proxy-mode=ipvs ?
Maybe then better to put that logic to getKubeproxyMode() ?